content/resources/hexagon/1.6.0/hexagon.core.css

```diff
@@ -8,7 +8,7 @@
  
  ----------------------------------------------------
  
- Version: 1.6.0
```

Comment:
+1 - The changes in `content/resources` should only be commited when the new version of hexagon is released (another argument for merging the two repos :) )